README.md

```diff
@@ -33,19 +33,15 @@ make build_deploy
 
 ### Updating Pre-installed Apps
 
-The names of pre-installed apps are specified in
-`pkg/misc/preinstalled.list`. These names are resolved to their most
-recent download targets, which are stored in
-`pkg/misc/preinstalled.list.lock`. Every so often, these can be updated
-so that new installs give the most recent versions:
+The names of pre-installed apps are specified in `settings.ini`. These names
+are resolved to their most recent download targets, or to the version specified.
+Every so often, these can be updated so that new installs give the most recent
+versions:
 
-This process relies on the `xpath` command in the `libxml-xpath-perl`
-package.
+This process relies on the `xpath` command in the `libxml-xpath-perl` package.
 
 ```
 sudo apt-get install libxml-xpath-perl
-make get_app_index
-make generate_applist_lockfile
```

Comment:
resolving with xpath is pretty slow, and when I'm building a lot, I do miss the speediness. We can merge this as-is, but I'll later have to add a way to skip the xpath stuff, ideally generating the lockfile once in advance